eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -131,6 +131,7 @@ jobs:
           pwsh: true
           workingDirectory: $(Build.SourcesDirectory)
           filePath: eng/scripts/SetTestPipelineVersion.ps1
+          arguments: '-BuildNumber $(Build.BuildNumber)'
```

Comment:
Did you mean to use BuildNumber or BuildID?

Reply:
Am using build number here because it easier to understand and its still unique as its the current date plus a incrementing number appened.